platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-haliburton.init

```diff
@@ -54,8 +54,8 @@ start)
         echo max6699 0x1a > /sys/bus/i2c/devices/i2c-11/new_device
 
         # Attach PSUs
-        echo dps200 0x5a > /sys/bus/i2c/devices/i2c-12/new_device
-        echo dps200 0x5b > /sys/bus/i2c/devices/i2c-13/new_device
+        # echo dps200 0x5a > /sys/bus/i2c/devices/i2c-12/new_device
+        # echo dps200 0x5b > /sys/bus/i2c/devices/i2c-13/new_device
```

Comment:
Remove commented code or add a `TODO` comment if you plan to uncomment later

Reply:
updated: https://github.com/Azure/sonic-buildimage/pull/6493/commits/0664a4fb56d29fa4fd8cb430a11a487822480abd